kolena/_experimental/classification/workflow.py

```diff
@@ -0,0 +1,247 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import dataclasses
+from typing import List
+from typing import Optional
+
+from pydantic.dataclasses import dataclass
+
+from kolena.workflow import define_workflow
+from kolena.workflow import EvaluatorConfiguration
+from kolena.workflow import GroundTruth as BaseGroundTruth
+from kolena.workflow import Image
+from kolena.workflow import Inference as BaseInference
+from kolena.workflow import Metadata
+from kolena.workflow import MetricsTestCase
+from kolena.workflow import MetricsTestSample
+from kolena.workflow import MetricsTestSuite
+from kolena.workflow.annotation import ClassificationLabel
+from kolena.workflow.annotation import ScoredClassificationLabel
+
+
+@dataclass(frozen=True)
+class TestSample(Image):
+    """Test sample type for the pre-built Classification workflow."""
+
+    metadata: Metadata = dataclasses.field(default_factory=dict)
+    """The metadata associated with the test sample."""
+
+
+@dataclass(frozen=True)
+class GroundTruth(BaseGroundTruth):
+    """Ground truth type for the pre-built Classification workflow."""
+
+    classification: ClassificationLabel
```

Comment:
We should think about whether we want to support a multi-label multiclass classification.
And for a binary classification, will this `classification` have to be something like this, (e.g., "dog" and "not dog")?

Reply:
`multi-label` should probably be a separate workflow.
For binary classification, users should be able to have the positive class be an optional `ClassificationLabel` which should be `None` for a negative class.